applications/longhorn/Kubefile

```diff
@@ -0,0 +1,4 @@
+FROM kubernetes:v1.19.9
```

Comment:
hi @mahmut-Abi ,good job, here i suggest we use "kubernetes:v1.19.8" as base image. "kubernetes:v1.19.8"  has passed fully test by our team.

Reply:
OK, I'll notice. I'll mention commit again this time. I'll change it and upload it in the future. Don't bother you. Thank you.